profitbricks/src/main/java/org/jclouds/profitbricks/domain/LoadBalancer.java

```
@@ -0,0 +1,398 @@
+
+         public abstract String ip();
+
+         @Nullable
+         public abstract String lanId();
+
+         @Nullable
+         public abstract List<String> serverIds();
+
+         public static CreatePayload create(String dataCenterId, String loadBalancerName, LoadbalancerAlgorithm loadBalancerAlgorithm, String ip, String lanId, List<String> serverIds) {
+            return new AutoValue_LoadBalancer_Request_CreatePayload(dataCenterId, loadBalancerName, loadBalancerAlgorithm, ip, lanId, serverIds);
+         }
+
+         public static class Builder {
+
+            public String dataCenterId;
+            public String loadBalancerName;
+            public LoadbalancerAlgorithm loadBalancerAlgorithm;
+            public String ip;
+            public String lanId;
+            public List<String> serverIds;
+
+            public Builder dataCenterId(String dataCenterId) {
+               this.dataCenterId = dataCenterId;
+               return this;
+            }
+
+            public Builder loadBalancerName(String loadBalancerName) {
+               this.loadBalancerName = loadBalancerName;
+               return this;
+            }
+
+            public Builder loadBalancerAlgorithm(LoadbalancerAlgorithm loadBalancerAlgorithm) {
+               this.loadBalancerAlgorithm = loadBalancerAlgorithm;
+               return this;
+            }
+
+            public Builder ip(String ip) {
+               this.ip = ip;
+               return this;
+            }
+
+            public Builder lanId(String lanId) {
+               this.lanId = lanId;
+               return this;
+            }
+
+            public Builder serverIds(List<String> serverIds) {
+               this.serverIds = serverIds;
+               return this;
+            }
+
+            public CreatePayload build() {
+               return CreatePayload.create(dataCenterId, loadBalancerName, loadBalancerAlgorithm, ip, lanId, serverIds);
```

Add the IP validation here as well. (So that we can preemptively fail users attempting to provision with invalid IP)